api/provider/aws/defaults.go

```diff
@@ -121,3 +124,19 @@ const DEFAULT_LB_ROLE_POLICY_TEMPLATE = `{
         }
     ]
 }`
+
+var DefaultCreateLBPortModel = []models.Port{
```

Comment:
I'm still up in the air about having a default port - it just seems 'lucky' that someone would need `80:80/tcp` for their service. If we do keep it, it should probably be just `var DefaultLBPort = models.Port`, not a slice (for easy conversion in regards to string in the cli and elb listeners objects in unit tests)

Reply:
after discussion, I think we should keep a default port due to the security group requirements. However, I still think this should be a single object, not a slice. Also, variable name is over-specific - 
```
var DefaultLoadBalancerPort = models.Port{ ... }
```

Reply:
Based on our previous conversation, because `CreateLoadBalancerInput` requires that you provide at least one `Listener` in a slice, we're opting to keep the notion of a default port unless we figure out a better UX pattern when a user wants to create a load balancer without providing a port.